ocaml/xapi/xapi_vgpu_type.ml

```diff
@@ -688,7 +691,7 @@ module Vendor_nvidia = struct
       max_resolution_x = conf.max_x;
       max_resolution_y = conf.max_y;
       size = Int64.div Constants.pgpu_default_size conf.max_instance;
-      internal_config = [Xapi_globs.vgpu_config_key, conf.file_path];
+      internal_config = [Xapi_globs.vgpu_config_key, conf.file_path; "type_id", conf.type_id];
```

Comment:
We don't need the config file path anymore, so remove it from here, and remove the key from xapi_globs. But add a value to xapi_globs for the type_id.

Reply:
OK, got it.